File: DSA_Questions/common_dsa_questions/common_dsa_questions/MinimumInArray.java
package common_dsa_questions;

public class MinimumInArray {

	public static void main(String[] args) {
		
		int[] arr = {1,2,3,4,5};
		
		int n = arr.length;
		
		int min=10000000;
		
	    for(var i=0; i<n; i++)
	    {
	        if(min>arr[i])
	        {
	            min = arr[i];
	        }
	    }
	    System.out.println(min);

	}

}

File: DSA_Questions/Python/MinimumInArray.py
n = 5
array = [-1, -5, 1, 2, 3, 4, 5]


def minimumInArray(N, arr):
    min_value = float("inf")
    for i in range(0, N):
        if arr[i] < min_value:
            min_value = arr[i]
    print(min_value)


minimumInArray(n, array)
